x2gbfs/util.py

```diff
@@ -38,3 +41,27 @@ def get(
     response = requests.get(url, headers=request_headers, timeout=timeout, params=params)
     response.raise_for_status()
     return response
+
+
+def post(
+    url: str,
+    params: Optional[dict[str, str]] = None,
+    headers: Optional[dict[str, str]] = None,
+    json: Optional[Dict] = None,
+    timeout: int = 5,
+    user_agent: str = 'x2gbfs +https://github.com/mobidata-bw/',
+):
+    request_headers = dict(headers) if headers is not None else {}
+    request_headers['User-Agent'] = user_agent
+    response = requests.post(url, headers=request_headers, timeout=timeout, params=params, json=json)
+    response.raise_for_status()
+    return response
+
+
+def reverse_multipolygon(geometry):
+    if geometry and geometry.get('type') == 'MultiPolygon':
+        for single_polygon in geometry.get('coordinates', []):
```

Comment:
Shall we assert – or at least document the assumption – that it's actually just *one* polygon?

Reply:
The code asserts that it's a MultiPolygon and iterates over all it's polygons. Maybe  `single_polygon` (for one of the multiple polygons) is missleading you?

Reply:
If there are multiple polygons, is there a non-limited service area, whose coordinates *should not* be reversed?

Reply:
No, there are not. GBFSv2 uses a non-spec-conformant GeoJSON style to address areas surrounding the operational area. I think we can't do much about it in GBFSv2 but try to migrate to GBFSv3, which has a much cleaner way of handling out-of-service-area areas. (using standard GeoJSON and defining default rules for areas not covered by a any polygon)